src/pkg/k8s.go

```diff
@@ -247,7 +247,13 @@ func (s *JobRunner) getPodObject(identifier string, labels map[string]string, jo
 // TODO: Remove all usages of "Viper" they should be passed in at JobRunner configuration time
 func (s *JobRunner) Run(job opslevel.RunnerJob, stdout, stderr *SafeBuffer) JobOutcome {
 	id := string(job.Id)
-	identifier := fmt.Sprintf("opslevel-job-%s-%d", job.Number(), time.Now().Unix())
+	var identifier string
+	switch viper.GetString("mode") {
+	case "faktory":
+		identifier = fmt.Sprintf("opslevel-job-%s-%d", job.Id, time.Now().Unix())
```

Comment:
this is a bit risky but @patrick-th and @BigBlue79 assure me they are always going to put through a number rather then a base64 hash......right guys!!! ;) 

Once we get off "the old API" method of runner we can circle back around to this and fix it.

Reply:
added